newbank/server/NewBank.java

```diff
@@ -89,6 +101,16 @@ private String addNewAccount(CustomerID customerID, List<String> request) {
     }
     return result;
   }
+  
+  private String listCommands(ArrayList<String> commands) {
+    String printCommands = new String();
+    for (String command : commands) {
+      printCommands += command;
+      printCommands += "\n";
+    }
+    printCommands = printCommands.substring(0, printCommands.length()-1);
+    return printCommands;

```

Comment:
Could simplify to:
return printCommands.substring(0, printCommands.length()-1);